modules/gittar/conf/conf.go

```diff
@@ -22,7 +22,7 @@ import (
 
 // Conf 定义配置对象.
 type Conf struct {
-	RepoRoot        string `env:"GITTAR_REPOSITORY_ROOT" default:"./repository"`
+	RepoRoot        string `env:"GITTAR_REPOSITORY_ROOT" default:"/repository"`
```

Comment:
check bind in version's dice.yml

Reply:
![image](https://user-images.githubusercontent.com/35135086/115485470-685ad500-a287-11eb-979e-288ac55ed87b.png)

yeah, it was mounted on the root path